internal/backends/postgresql/backend.go

```diff
@@ -128,8 +128,9 @@ func (b *backend) ListDatabases(ctx context.Context, params *backends.ListDataba
 		if err != nil {
 			return nil, lazyerrors.Error(err)
 		}
-
-		stats, err := db.Stats(ctx, new(backends.DatabaseStatsParams))
+		dbStatParams := new(backends.DatabaseStatsParams)
+		dbStatParams.Refresh = true
```

Comment:
Why we do that?

Reply:
It was a temporary change , to call dbStat and CollStat with refresh: true.

Reply:
I think it is better to add to this test: https://github.com/FerretDB/FerretDB/blob/2d57d3ff41bc6cf09a6d82d71424bee864b520e7/internal/backends/collection_test.go#L136

Reply:
I understand now , thank you 🙂